packages/manager/cypress/e2e/core/cloudpulse/cloudpulse-applied-filters.spec.ts

```diff
@@ -283,11 +301,26 @@ describe('Integration Tests for Applied Filters', () => {
     //Collapse the Filters section
     ui.button.findByTitle('Filters').should('be.visible').click();
     cy.get('[data-testid="applied-filter"]').within(() => {
-      cy.get(`[data-qa-value="Database Engine PostgreSQL"]`)
+      cy.get(`[data-qa-value="Database Engine ${'PostgreSQL'}"]`)
+        .should('be.visible')
+        .should('have.text', 'PostgreSQL');
+
       cy.get(`[data-qa-value="Region US, Chicago, IL"]`)
-      cy.get(`[data-qa-value="Node Type Primary"]`)
-      cy.get(`[data-qa-value="Database Clusters ${databaseMock.label}"]`)
-      cy.get(`[data-qa-value="Database Clusters ${extendDatabaseMock.label}"]`)
+        .should('be.visible')
+        .should('have.text', 'US, Chicago, IL');
+
+      cy.get(`[data-qa-value="Node Type ${'Primary'}"]`)
```

Comment:
@agorthi-akamai , here no need for back ticks right

Reply:
@venkymano-akamai , i have addressed  it :)